src/test/java/org/sagebionetworks/bridge/dynamodb/DynamoAppConfigTest.java

```diff
@@ -148,6 +157,12 @@ public void canSerialize() throws Exception {
                 appConfig.getSurveyReferences().get(1).getGuid());
         assertEquals(node.get("surveyReferences").get(1).get("createdOn").textValue(), SURVEY_PUB_DATE.toString());
         
+        assertEquals(node.get("fileReferences").size(), 2);
+        assertEquals(node.get("fileReferences").get(0).get("guid").textValue(), GUID);
+        assertEquals(node.get("fileReferences").get(0).get("createdOn").textValue(), TIMESTAMP.toString());
+        assertEquals(node.get("fileReferences").get(1).get("guid").textValue(), "twoGuid");
+        assertEquals(node.get("fileReferences").get(1).get("createdOn").textValue(), TIMESTAMP.toString());
+        
         AppConfig deser = BridgeObjectMapper.get().treeToValue(node, AppConfig.class);
         assertNull(deser.getStudyId());
```

Comment:
Should we check file references in the deserialized object?

Reply:
Added